apitools/gen/util.py

```diff
@@ -373,7 +388,16 @@ def FetchDiscoveryDoc(discovery_url, retries=5):
     for url in discovery_urls:
         for _ in range(retries):
             try:
-                content = urllib_request.urlopen(url).read()
+                response = urllib_request.urlopen(url)
```

Comment:
Can we pull the request code into a method and add tests for the possible content encodings? 

Something similar to https://github.com/httplib2/httplib2/blob/master/tests/test_encoding.py

I was surprised that deflate is handled the same way as no encoding.

Reply:
Done.